src/coinbase/address/wallet_address.ts

```diff
@@ -752,41 +754,62 @@ export class WalletAddress extends Address {
   /**
    * Fund the address from your account on the Coinbase Platform.
    *
-   * @param amount - The amount of the Asset to fund the wallet with
-   * @param assetId - The ID of the Asset to fund with. For Ether, eth, gwei, and wei are supported.
+   * @param options - The options to create the fund operation
+   * @param options.amount - The amount of the Asset to fund the wallet with
+   * @param options.assetId - The ID of the Asset to fund with. For Ether, eth, gwei, and wei are supported.
    * @returns The created fund operation object
    */
-  public async fund(amount: Amount, assetId: string): Promise<FundOperation> {
-    const normalizedAmount = new Decimal(amount.toString());
+  public async fund(options: CreateFundOptions): Promise<FundOperation> {
```

Comment:
```suggestion
  public async fund({ assetId, amount }: CreateFundOptions)
```

Can we dereference the parameter like in the other functions:
https://github.com/coinbase/coinbase-sdk-nodejs/blob/5c0a2a6735d8ca96dba8f870814acf07120d1bb5/src/coinbase/address/wallet_address.ts#L208-L213

Reply:
seems we do this for some functions and not others in wallet_address? e.g. deployToken doesn't dereference. in any case i can update all to dereference!